### ISSUE TITLE
Message list: Separate lines for sender and message preview

### ISSUE THREAD

**Issue by uniquePWD** [thunderbird/thunderbird-android#6733]
### Checklist

- [X] I have used the search function to see if someone else has already submitted the same feature request.
- [X] I will describe the problem with as much detail as possible.
- [X] This issue only contains a request for one single feature, **not** multiple (related) features.

### App version

6.509

### Problem you are trying to solve

Currently and previously, K9 gave a wall of text vibe when you tried to scan the message list. This can be fixed by simply moving the message preview to a new line. 

### Suggested solution

Line 1: Subject
Line 2: Sender 
Line 3+: Preview Text

### Screenshots / Drawings / Technical details

![Screenshot_20230307-174917](https://user-images.githubusercontent.com/203217/223506483-b99ab768-69b4-433e-a107-5895650356fb.png)

**Comment by rms-joe** [comment]
@pwd-github there's already a setting to achieve something close to your suggestiin: settings>general>display>correspondant above subject.

though i agree on the wall of of text statement. actually i was lookin forward to the newly introduced high density message view of 6.509, but roll back as subject now even has same color as message preview after marked read, due to unusable wall of text perception.

proposal/idea: change subject from redor blue to black/white to show read/unread status. (similar to how thunderbird new multiline message view gonna do, from blue to black/white from what i saw on preview pics)

**Comment by ChristopherBull** [comment]
This would make the list easier to read and would follow similar apps that have a better message list UI/UX (Outlook, Edison), in my opinion.

**Comment by drm-nz** [comment]
I agree with the original idea. The wall of text look is true even with "Correspondent above subject" enabled. We could potentially keep the current look for Compact, and perhaps even for Default, but I think this would be a welcome change by many for the Relaxed density.

**Comment by Alecaddd** [comment]
Thank you for the report.
Indeed, the subject and message preview don't have a good visual treatment and they're hard to distinguish.
I'm gonna close this bug as duplicate of #10407 as we're rebuilding everything from scratch and introducing a much more well considered and powerful UI for the message list.
After that implementation is done, please feel free to report other specific enhancement if needed.
Cheers
